lab2.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 26 10:23:42 2020
+
+@author: User
+"""
+
+
+
+
+from functools import reduce
+import numpy 
+import time  
+def unpackTuple(stream): 
+    reduce(numpy.append, stream) 
+stream = ([0,4,6,7,6,3,4,6,1,2,7] , 5 )   
+#stream = ([9,1,1,1,0,0,0,8,10,11,14,2132323,24234524], 2)     
+unpackTuple(stream)
+
+def threshold_bubble(lista, prog):
+    greater3 = []
+    i = 0
+    while len(greater3) < 3:
+        if lista[i] > prog:
+            greater3.append(i)
+        i += 1
+    
+    
+    for i in range(0, len(greater3)):
+        for j in range(len(greater3)-1):
+            if lista[greater3[j]]<lista[greater3[j+1]]:
+                t = greater3[j]
+                greater3[j] = greater3[j+1]
+                greater3[j+1] = t
+    return greater3
+
+
+
+
+def threshold_insert(lista,prog):
+    greater3 = []
```

Comment:
lepiej byłoby to nazwać result a nazwę funkcji nazwać lepiej- tak by wiadomo było co ona zwraca, co liczy